### ISSUE TITLE
Weird code block rendering

### ISSUE THREAD

**Issue by jsamr** [Rebilly/ReDoc#611]
**Version**: redoc@2.0.0-alpha.36, redoc-cli@0.6.1  
**Tested browsers**:

- Firefox 61.0.2 (64-bit)
- Chromium 68.0.3440.106 (64-bit)

### Markdown source code

This markdown is extracted from `info.description`

<pre>
``` text
  Link: &lt;/protocoles/18/sections/34&gt; rel=backlink; title*=UTF-8'fr'Accident%20d%27exposition%20au%20sang,
        &lt;/techniques/65/sections/45&gt; rel=backlink; title*=UTF-8'fr'Valve%20ASHERMAN
```
</pre>

### Expected visual output

![good-rendered](https://user-images.githubusercontent.com/3646758/44354885-08c61f00-a479-11e8-9dfa-89fccf4be3cd.png)

**This visual output disappears after few milliseconds**.

### Observed visual output

![bad-rendered](https://user-images.githubusercontent.com/3646758/44354899-1085c380-a479-11e8-917b-a4b94db69db3.png)

### Observed DOM

``` html
<code class=" language-json">Link: &lt;/protocoles/18/sections/34&gt; rel=backlink; title*=UTF-8'fr'Accident%20d%27exposition%20au%20sang,
      &lt;/techniques/65/sections/45&gt; rel=backlink; title*=UTF-8'fr'Valve%20ASHERMAN
</code>
```

My guess is that a class is removed with some sort of sanitizing process, resulting in an ugly code block.

**Comment by RomanGotsiy** [comment]
Should be fixed with `redoc@2.0.0-alpha.37`
Could you verify please?

**Comment by jsamr** [comment]
@RomanGotsiy Yes I can confirm, the issue has vanished! Thanks a lot.
